test-binaries/src/bin/peer-state.rs

```diff
@@ -0,0 +1,37 @@
+use ntp_daemon::ObservablePeerState;
+use std::error::Error;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
```

Comment:
just a test application for now. We can expand this into the full client but I didn't want to do command-line argument parsing in this PR